backend/cms/apps.py

```diff
@@ -1,12 +1,17 @@
 """
 Django related class representing a config of an app
 """
+import sys
+from django.conf import settings
 from django.apps import AppConfig
 
-
 class CmsConfig(AppConfig):
     """
     Class inheriting the django AppConfig
     """
 
     name = 'cms'
+
+    if settings.SECRET_KEY == '-!v282$zj815_q@htaxcubylo)(l%a+k*-xi78hw*#s2@i86@_' and not settings.DEBUG:
```

Comment:
I'm not sure, but as [I understand the docs](https://docs.djangoproject.com/en/dev/ref/applications/#django.apps.AppConfig.ready), we need a function which is called `ready`, and place the check inside this function...

Reply:
This indeed fixes the issue of multiple execution. And yes, probably better than having this in the class itself :sweat_smile: